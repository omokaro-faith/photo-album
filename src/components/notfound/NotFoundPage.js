import React from 'react';
import { Link } from 'react-router-dom';
import '../../styles/components/notfound/_notfound.css'
const NotFoundPage = () => (
  <div>
    <h1>Nothing to see here, go back to <Link to='/'>albums page </Link></h1>
  </div>
)

 export default NotFoundPage; 