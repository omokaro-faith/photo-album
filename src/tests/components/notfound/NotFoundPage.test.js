import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../../components/notfound/NotFoundPage';

describe('NotFoundPage', () => {
  let props;
  let wrapper;

  beforeEach(() => {
    props = { 
      totalItems: 2,
      pageName: 'Photos',
    }
    wrapper = shallow(<NotFoundPage {...props} />);
  });

  afterEach(() => { wrapper = null; });

  
  describe('renders', () => {
    it('should exist', () => {
      expect(wrapper.exists()).toBe(true);
    });

    it('should not have any regressions', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
