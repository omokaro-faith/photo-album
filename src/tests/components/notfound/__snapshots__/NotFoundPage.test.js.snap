// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders should not have any regressions 1`] = `ShallowWrapper {}`;
